### ISSUE TITLE
Exception in StyleguideRouter

### ISSUE THREAD

**Issue by infabo** [sitegeist/fluid-styleguide#79]
Visit: `/fluid-styleguide/list`

```
Call to undefined method TYPO3\CMS\Extbase\Mvc\Web\Request::withAttribute()
```

![image](https://user-images.githubusercontent.com/3999104/153257895-bc37a9b5-a094-4065-991a-22b640aa3675.png)

* TYPO3 10.4.24
* ext:fluid_styleguide 2.0.1

**Comment by infabo** [comment]
I see this has been fixed already in 7fd06e89323de6283358149b8d8f04ee7a321772

Should do a release, huh? ;)

**Comment by s2b** [comment]
I just published v2.0.2.
